index.js

```diff
@@ -1 +1,88 @@
 // implement your API here
+//
+const express = require('express');
+
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json()); 
+
+server.post('/users', (req, res) => {
+	const userInfo = req.body;
```

Comment:
Consider destructuring:

```suggestion
	const {name, bio} = req.body;
```